examples/node-app/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "scripts": {
+    "start": "node server.js",
+    "start-local": "WEBPACK_ENV=es6 node server.js"
+  },
+  "dependencies": {
+    "express": "^4.17.1",
+    "global": "^4.3.0",
+    "kepler.gl": "^1.1.5",
```

Comment:
`^1.1.6`

Reply:
I published version 1.1.6 and updated the file